src/main/java/com/codeborne/selenide/proxy/SelenideProxyServer.java

```diff
@@ -143,7 +143,7 @@ static InetSocketAddress getProxyAddress(Proxy proxy) {
   @CheckReturnValue
   @Nonnull
   private Proxy createSeleniumProxy() {
-    return isEmpty(config.proxyHost())
+    return config.proxyHost() == null
```

Comment:
I don't see  why this change is useful. 
if `config.proxyHost()` is empty string, the following line will throw an exception anyway.

Reply:
@BorisOsipov ^^^